core/src/main/scala/com/karumi/shot/android/Adb.scala

```diff
@@ -11,6 +11,10 @@ object Adb {
 class Adb {
 
   private final val CR_ASCII_DECIMAL = 13
+  private val logger = ProcessLogger(
+    o => println("Shot ADB output: " + o),
+    e => println(Console.RED + "Shot ADB error: " + e + Console.RESET)
```

Comment:
Same as `output` I'd change it to `error`

Reply:
I don't understand you suggestion

Reply:
```suggestion
    error => println(Console.RED + "Shot ADB error: " + error + Console.RESET)
```

Reply:
Fixed!